pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -183,29 +183,31 @@ func NewManager(policyName string, machineInfo *cadvisorapi.MachineInfo, nodeAll
 
 // Start starts the memory manager under the kubelet and calls policy start
 func (m *manager) Start(activePods ActivePodsFunc, sourcesReady config.SourcesReady, podStatusProvider status.PodStatusProvider, containerRuntime runtimeService, initialContainers containermap.ContainerMap) error {
-	klog.InfoS("Starting memorymanager", "policy", m.policy.Name())
+	ctx := context.TODO()
```

Comment:
Is it possible to pass context into this function?

Reply:
Yep, going to try to do that :)

Reply:
@bart0sh I have tried to address this comment of yours. Please take a look at this commit: https://github.com/kubernetes/kubernetes/pull/130727/commits/80ca4d187d5d9f882744d86f4269da73695905e1 and let me know your thoughts. It is worth having a discussion about this to make sure we are happy to proceed this way as we should stay consistent across resource managers.

Reply:
I will squash the commits once we are happy with the approach proposed in the commit.